builders/ethereum/dev-env/thirdweb.md

```diff
@@ -110,184 +112,339 @@ thirdweb offers SDKs for a range of programming languages, such as React, React
+
+### Accounts and Wallets
+
+thirdweb distinguishes between accounts and wallets in the sdk. In the eyes of the thirdweb SDK, an account always has a single blockchain address and can sign messages, transactions, and typed data, but it cannot be "connected" or "disconnected." In contrast, a wallet contains one or more accounts, can be connected or disconnected, and delegates the signing tasks to its accounts. 
+
+The below code snippet demonstrates how to initialize and connect a MetaMask wallet using the thirdweb SDK, then sign and send a transaction, retrieving the transaction hash. This process is applicable to any of the 300+ wallet connectors supported by the SDK.
+
+```typescript title="Initialize and Connect a Metamask Wallet"
+import { sendTransaction } from 'thirdweb';
+// MetaMask wallet used for example, the pattern is the same for all wallets
+import { createWallet } from 'thirdweb/wallets';
+
+// initialize the wallet, Third web supports of the 300+ wallet connectors
+const wallet = createWallet('io.metamask');
+
+// connect the wallet, this returns a promise that resolves to the connected account
+const account = await wallet.connect({
+  // pass the client you created with `createThirdwebClient()`
+  client,
+});
+
+// sign & send a transaction with the account -> returns the transaction hash
+const { transactionHash } = await sendTransaction({
+  // assuming you have called `prepareTransaction()` or `prepareContractCall()` before which returns the prepared transaction to send
+  transaction,
+  // Pass the account to sign the transaction with
+  account,
+});
 ```
 
 ### Get Contract {: #get-contract }
 
-To connect to your contract, use the SDK’s [`getContract`](https://portal.thirdweb.com/typescript/sdk.thirdwebsdk.getcontract){target=\_blank} method.
+To connect to your contract, use the SDK’s [`getContract`](https://portal.thirdweb.com/references/typescript/v5/getContract){target=\_blank} method.  As an example, you could fetch data from an [incrementer contract on Moonbase Alpha](https://moonbase.moonscan.io/address/0xa72f549a1a12b9b49f30a7f3aeb1f4e96389c5d8){target=\_blank}.
 
-```javascript
-import { useContract } from '@thirdweb-dev/react';
+```typescript title="Get Contract"
+import { getContract } from 'thirdweb';
+import { client } from './client';
 
-function App() {
-  const { contract, isLoading, error } = useContract('INSERT_CONTRACT_ADDRESS');
-}
+const myContract = getContract({
+  client,
+  chain: moonbase,
+  address: INSERT_CONTRACT_ADDRESS,
+  abi: INSERT_ABI,
```

Comment:
If we're going to tell them they can fetch data from an incrementer contract, we should just put the contract address and ABI in the code snippet